doc/sphinx/changes.rst

```diff
@@ -8,6 +8,687 @@ Recent changes
+  (`#13381 <https://github.com/coq/coq/pull/13381>`_,
+  by Jim Fehrle).
+- **Added:**
+  :tacn:`lia` is extended to deal with boolean operators e.g. `andb` or `Z.leb`.
+  (As `lia` gets more powerful, this may break proof scripts relying on `lia` failure.)
+  (`#11906 <https://github.com/coq/coq/pull/11906>`_,  by Frédéric Besson).
+- **Added:**
+  :tacn:`apply … in` supports several hypotheses
+  (`#12246 <https://github.com/coq/coq/pull/12246>`_,
+  by Hugo Herbelin; grants
+  `#9816 <https://github.com/coq/coq/pull/9816>`_).
+- **Added:**
+  The :tacn:`zify` tactic can now be extended by redefining the `zify_pre_hook`
+  tactic. (`#12552 <https://github.com/coq/coq/pull/12552>`_,
+  by Kazuhiko Sakaguchi).
+- **Added:**
+  The :tacn:`zify` tactic provides support for primitive integers (module :g:`ZifyInt63`).
+  (`#12648 <https://github.com/coq/coq/pull/12648>`_,  by Frédéric Besson).
+- **Fixed:**
+  Avoid exposing an internal name of the form :n:`_tmp` when applying the
+  :n:`_` introduction pattern which would break a dependency
+  (`#13337 <https://github.com/coq/coq/pull/13337>`_,
+  fixes `#13336 <https://github.com/coq/coq/issues/13336>`_,
+  by Hugo Herbelin).
+- **Fixed:**
+  The case of tactics, such as :tacn:`eapply`, producing existential variables
+  under binders with an ill-formed instance
+  (`#13373 <https://github.com/coq/coq/pull/13373>`_,
+  fixes `#13363 <https://github.com/coq/coq/issues/13363>`_,
+  by Hugo Herbelin).
+
+Tactic language
+^^^^^^^^^^^^^^^
+
+- **Added:**
+  An if-then-else syntax to Ltac2
+  (`#13232 <https://github.com/coq/coq/pull/13232>`_,
+  fixes `#10110 <https://github.com/coq/coq/issues/10110>`_,
+  by Pierre-Marie Pédrot).
+- **Fixed:**
+  Printing of the quotation qualifiers when printing :g:`Ltac` functions
+  (`#13028 <https://github.com/coq/coq/pull/13028>`_,
+  fixes `#9716 <https://github.com/coq/coq/issues/9716>`_
+  and `#13004 <https://github.com/coq/coq/issues/13004>`_,
+  by Hugo Herbelin).
+
+SSReflect
+^^^^^^^^^
+
+- **Added:**
+  SSReflect intro pattern ltac views ``/[dup]``, ``/[swap]`` and ``/[apply]``
+  (`#13317 <https://github.com/coq/coq/pull/13317>`_,
+  by Cyril Cohen).
+- **Fixed:**
+  Working around a bug of interaction between + and /(ltac:(...)) cf
+  `#13458 <https://github.com/coq/coq/issues/13458>`_
+  (`#13459 <https://github.com/coq/coq/pull/13459>`_,
+  by Cyril Cohen).
+
+Commands and options
+^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  When compiled with OCaml >= 4.10.0, Coq will use the new best-fit GC
+  policy, which should provide some performance benefits. Coq's policy
+  is optimized for speed, but could increase memory consumption in
+  some cases. You are welcome to tune it using the ``OCAMLRUNPARAM``
+  variable and report back on good settings so we can improve the defaults.
+  (`#13040 <https://github.com/coq/coq/pull/13040>`_,
+  fixes `#11277 <https://github.com/coq/coq/issues/11277>`_,
+  by Emilio Jesus Gallego Arias).
```

Comment:
This entry is misplaced (probably due to the ambiguity of the name "Commands and options"). This entry should be in "Infrastructure and dependencies".

Reply:
Right.